tests/Behat/features/edit-block-element.feature

```diff
@@ -68,7 +68,7 @@ Feature: Edit elements in the CMS
       And I fill in "Charlie's Block" for "Title" for block 2
     When I press the "Save" button in the actions menu for block 2
       And I wait 1 second
-      Then I should see a "Saved 'Charlie's Block' successfully" notice
+      Then I should see a "Saved 'Charlie's Block' successfully" success toast
```

Comment:
The "I should see some notice" fixture was removed, as it had previously been deprecated. See [silverstripe/silverstripe-framework#10594](https://github.com/silverstripe/silverstripe-framework/pull/10594/files#diff-b5ac2578edc2e46f01f689ceab76303bb7c8897d79e80962a1d6d65e18db7595L79-L87)